src/index.css

```diff
@@ -1,13 +1,105 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap");
+
+/* base styles */
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: "Quicksand";
+  /* color: #333; */
+}
+.navbar {
```

Comment:
don't add styling to index.css. Only add theme related styling here. Component specific styling should be done like we discussed: styled components. We can go over it again on call.

Reply:
Done